go/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.1
 
 require (
 	github.com/PuerkitoBio/goquery v1.9.1
-	github.com/playwright-community/playwright-go v0.4201.1
```

Comment:
Is this leading to a never-ending sequence of manual version updating?  Can't we just specify that we want the latest?

Reply:
I'm not sure. I think I hit https://github.com/playwright-community/playwright-go/issues/409 when trying it out.

What the project really needs is a Github action to verify the test.